### DISCUSSION TITLE
Failed to download from mirror set

### DISCUSSION THREAD

**Discussion by UniverGao** [microsoft/vcpkg#29042] [Q&A]
****

**Is your feature request related to a problem? Please describe.**
I use computer in company and IT department support Internet support via proxy. But the certification of web should be replaced by the proxy server, so the procedure come to fail when I use vcpkg to download the packs, the error message is shown below:
`error: Failed to download from mirror set`
`error: https://nodejs.org/dist/v16.15.1/node-v16.15.1-linux-x64.tar.gz: curl failed to download with exit code 60`

**Proposed solution**
Close certification identify of curl command

**Describe alternatives you've considered**
Null

**Additional context**
Null

**Comment by JackBoosY** [comment]
I assume you are a Chinese: https://gitee.com/jackboosy/vcpkg/issues/I44496
